src/index.js

```diff
@@ -15,17 +15,39 @@ const VueTribute = {
   },
   mounted() {
     const $el = this.$slots.default[0].elm
+    this.createTributeInstance($el, this.options)
+  },
+  watch: {
+    options: {
+      handler: function (newVal) {
+        this.$nextTick(()=>{ // recreate the tribute instance for newest options
```

Comment:
Can you fix the formatting here? Should look more like `this.$nextTick(() => {`

Reply:
Sorry, this is my mistake, I will fix it soon.